database/schema.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE labels (
+  id SERIAL PRIMARY KEY,
+  title VARCHAR(200) NOT NULL,
+  color VARCHAR(200) NOT NULL
+)
+
+CREATE TABLE books (
+  id SERIAL PRIMARY KEY,
+  publish_date DATE NOT NULL,
+  archived BOOLEAN NOT NULL,
+  genre_id INT,
+  author_id INT,
+  label_id INT,
+  FOREIGN KEY (genre_id) REFERENCES genres(id),
+  FOREIGN KEY (author_id) REFERENCES authors(id),
+  FOREIGN KEY (label_id) REFERENCES labels(id)  
+)
```

Comment:
-Kindly see this part of instruction Create a schema.sql file with tables that will be analogical to the structure of the classes in your app. As you cannot implement inheritance in the database tables - add all properties and associations from the parent Item class as table columns to all tables based on the child classes. Thanks to that it can be used to store data in the [future.](https://github.com/microverseinc/curriculum-ruby/blob/main/group-capstone/catalog_of_my_things.md)